Makefile

```diff
@@ -21,14 +21,20 @@ PASSED_MBEDTLS_CFLAGS := $(CFLAGS) -DCKB_DECLARATION_ONLY -I ../../ckb-c-stdlib-
 
 SECP256K1_SRC_20210801 := deps/secp256k1-20210801/src/ecmult_static_pre_context.h
 
+CFLAGS_LIBECC := -fno-builtin -DUSER_NN_BIT_LEN=256 -DWORDSIZE=64 -DWITH_STDLIB -DWITH_CKB -DCKB_DECLARATION_ONLY -fPIC -g -O3
```

Comment:
Why are -DWITH_STDLIB and -DWITH_CKB both enabled?

Reply:
WITH_STDLIB is defined upstream. Some ckb specific functions are enabled by WITH_CKB. To use a single WITH_CKB, quite a few upstream code need to be changed.